practices/risk_analysis.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: risk-analysis
+stages:
+  - development
+short_description: Risk Analysis and Management is a key project management practice to ensure that the least number of surprises occur while the project is underway
+tags:
+  - risk analysis
+  - software documentation
+keywords:
+  - risk analysis
+  - documentation
+  - task
+---
+
+# Risk Analysis
+
+**TL;DR**
+
+Risk analysis is the process of identifying and analyzing potential issues that could negatively impact key business initiatives or projects. This process is done in order to help organizations avoid or mitigate those risks.
+
+## What Is Risk Analysis
+
+Risk analysis is the process of analyzing, determining, and defining the risk of danger to government agencies, and businesses. We also use it to determine risk to individuals, towns, cities, and whole national economies.
```

Comment:
_Risk analysis is the process of analyzing, determining, and defining the risk of danger to government agencies, and businesses. We also use it to determine risk to individuals, towns, cities, and whole national economies._

_Specifically, the analysis looks at potential natural and anthropogenic events. Anthropogenic events are events that human activity causes._ from https://marketbusinessnews.com/financial-glossary/risk-analysis-definition-meaning/